Hymns/Common/SearchBar.swift

```diff
@@ -46,8 +46,8 @@ struct SearchBar: View {
                     }
                 }
                 .foregroundColor(Color(.systemBlue))
-                .transition(AnyTransition.move(edge: .trailing))
-                .animation(.easeInOut(duration: 0.2))
+                .transition(.customSearchTransition)
```

Comment:
like can we just have this:

`.transition(.asymmetric(insertion: .opacity, removal: .move(edge: .trailing)))`

Reply:
Nice, yeah much better thanks!